lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -1546,6 +1558,48 @@ class ByteCodeBufferBuilder
     }
 #endif
 
+    void TranslateDeferredStubTree(DeferredFunctionStub* deferredStubs, uint stubsCount, Recycler *recycler)
+    {
+        if (deferredStubs == nullptr || stubsCount == 0)
```

Comment:
Can we assert/fail-fast if stubsCount > 0 and deferredStubs == nullptr, or the inverse of that?

Reply:
Ok